vite.config.ts

```diff
@@ -18,5 +18,6 @@ export default defineConfig({
             provider: "v8",
         },
         environment: "happy-dom",
+        exclude: ["**/*.{yml}"],
```

Comment:
I guess you can use this pattern for this instead of adding exclude here

https://github.com/facebook/create-react-app/issues/1455#issuecomment-276004472

Reply:
I forgot to specify `node_modules` 😓